permission-portal-cloud-functions/firestore.rules

```diff
@@ -47,5 +47,9 @@ service cloud.firestore {
       allow read: if isSignedIn() && isInOrganization(organizationID);
       allow write: if isSignedIn() && isAdmin() && isInOrganization(organizationID);
     }
+
+    match /userImages/{email} {
+	    allow read, write: if isSignedIn() && isSelfOrAdminInOrg(email, resource.data.organizationID);
+    }
```

Comment:
i dont know if an admin is ever required to modify the profile images. it is typical for this rule to be isSelf only

Reply:
Sorry, missed this before I merged. You're right, admin doesn't need to be able to modify, will fix in a different PR